tensorflow/lite/micro/examples/hello_world/spresense/Makefile.inc

```diff
@@ -0,0 +1,14 @@
+ifeq ($(TARGET), spresense)
```

Comment:
Can you add a section to the README, explaining how a developer could set up and build this (and other) examples using the Spresense toolchain? If you can follow the format we use for other platforms like the ESP32/STM32/etc, that will give us the chance to test it too.